.github/workflows/noble.yml

```
@@ -19,18 +20,19 @@ jobs:
     - name: Get the list of packages
       run: |
         cp noble/packages_to_import /tmp/patched-packages
-    - name: Install Dependencies
+    - name: Install Dependencies and enable Sources
       run: |
+        sed -i 's/Types: deb/Types: deb deb-src/g' /etc/apt/sources.list.d/ubuntu.sources
```

```suggestion
        sed -i 's/^\(Types: deb\)$/\1 deb-src/g' /etc/apt/sources.list.d/ubuntu.sources
```

It wouldn't happen, but running the current sed multiple times always appends `deb-src`, so we should make sure not to append if it has already done. Also use a back-reference to DRY